src/main/java/tech/thatgravyboat/repolib/api/RecipesAPI.java

```diff
@@ -19,7 +19,8 @@ static RecipesAPI load(JsonElement json) {
         if (json instanceof JsonArray array) {
             for (var element : array) {
                 Recipe<?> recipe = Recipe.parse(element.getAsJsonObject());
-                recipes.computeIfAbsent(recipe.type(), k -> new ArrayList<>()).add(recipe);
+                if (recipe == null) continue;
+                recipes.computeIfAbsent(recipe.type(), _ -> new ArrayList<>()).add(recipe);
```

Comment:
sadly wrong java version, were on java 21 for this project

Reply:
it worked for me :sob: